packages/common/src/types/savedCharts.ts

```diff
@@ -22,8 +22,13 @@ export type BigNumberConfig = {
     config?: BigNumber;
 };
 
+export type ColumnProperties = {
+    field: string;
+    visible?: boolean;
+};
 export type TableChart = {
     showTableNames?: boolean;
+    columns?: ColumnProperties[];
```

Comment:
What you think if we used a map instead of an array since the fieldId is unique ? 
We would avoid all the looping to find the column we want.